llvm/lib/Analysis/MLInlineAdvisor.cpp

```diff
@@ -206,7 +206,6 @@ void MLInlineAdvisor::onPassEntry(LazyCallGraph::SCC *LastSCC) {
   // care about the nature of the Edge (call or ref). `FunctionLevels`-wise, we
   // record them at the same level as the original node (this is a choice, may
   // need revisiting).
-  NodeCount -= static_cast<int64_t>(NodesInLastSCC.size());
   while (!NodesInLastSCC.empty()) {
     const auto *N = *NodesInLastSCC.begin();
     NodesInLastSCC.erase(N);
```

Comment:
(this comment is meant for line 213, but github doesn't let me, so..) do you mean to `assert (!N->isDead())`?

Reply:
not in this PR, this doesn't change dead function handling